code/authenticators/LDAPLoginForm.php

```diff
@@ -22,6 +22,14 @@ class LDAPLoginForm extends MemberLoginForm
      */
     protected $ldapSecController = null;
 
+    /**
+     * Time in nanoseconds that we use to ensure consistent repsonse times - 1mil nanoseconds = 1 second
```

Comment:
soo.. I assume that you want to use `microseconds` and a there are  1 000 000 000 ns per seconds. So microseconds?

Reply:
upcoming commit changes this, it was dumb